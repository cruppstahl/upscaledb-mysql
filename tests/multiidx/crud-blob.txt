DROP TABLE IF EXISTS test;
CREATE TABLE test (value VARCHAR(30) NOT NULL, INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb;
INSERT INTO test VALUES("1", 1);
INSERT INTO test VALUES("1", 2);
INSERT INTO test VALUES("1", 3);
INSERT INTO test VALUES("2", 4);
INSERT INTO test VALUES("3", 5);
SELECT * FROM test;
SELECT * FROM test WHERE num >= 3;
SELECT * FROM test WHERE value = "1";
DELETE FROM test WHERE num = 1;
SELECT * FROM test;
DELETE FROM test WHERE value = "2";
SELECT * FROM test;
DELETE FROM test WHERE 1;
INSERT INTO test VALUES("1", 1);
INSERT INTO test VALUES("1", 2);
INSERT INTO test VALUES("1", 3);
INSERT INTO test VALUES("2", 4);
INSERT INTO test VALUES("3", 5);
UPDATE test SET num = 6 WHERE value = "3";
SELECT * FROM test;
UPDATE test SET value = "value" WHERE num = 1;
SELECT * FROM test;
UPDATE test SET value = "value" WHERE num <= 4;
SELECT * FROM test;
