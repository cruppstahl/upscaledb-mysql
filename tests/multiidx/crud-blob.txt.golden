--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (value VARCHAR(30) NOT NULL, INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES("1", 1)
--------------

--------------
INSERT INTO test VALUES("1", 2)
--------------

--------------
INSERT INTO test VALUES("1", 3)
--------------

--------------
INSERT INTO test VALUES("2", 4)
--------------

--------------
INSERT INTO test VALUES("3", 5)
--------------

--------------
SELECT * FROM test
--------------

value	num
1	1
1	2
1	3
2	4
3	5
--------------
SELECT * FROM test WHERE num >= 3
--------------

value	num
1	3
2	4
3	5
--------------
SELECT * FROM test WHERE value = "1"
--------------

value	num
1	1
1	2
1	3
--------------
DELETE FROM test WHERE num = 1
--------------

--------------
SELECT * FROM test
--------------

value	num
1	2
1	3
2	4
3	5
--------------
DELETE FROM test WHERE value = "2"
--------------

--------------
SELECT * FROM test
--------------

value	num
1	2
1	3
3	5
--------------
DELETE FROM test WHERE 1
--------------

--------------
INSERT INTO test VALUES("1", 1)
--------------

--------------
INSERT INTO test VALUES("1", 2)
--------------

--------------
INSERT INTO test VALUES("1", 3)
--------------

--------------
INSERT INTO test VALUES("2", 4)
--------------

--------------
INSERT INTO test VALUES("3", 5)
--------------

--------------
UPDATE test SET num = 6 WHERE value = "3"
--------------

--------------
SELECT * FROM test
--------------

value	num
1	1
1	2
1	3
2	4
3	6
--------------
UPDATE test SET value = "value" WHERE num = 1
--------------

--------------
SELECT * FROM test
--------------

value	num
value	1
1	2
1	3
2	4
3	6
--------------
UPDATE test SET value = "value" WHERE num <= 4
--------------

--------------
SELECT * FROM test
--------------

value	num
value	1
value	2
value	3
value	4
3	6
